java/src/jmri/jmrix/can/cbus/node/CbusNodeConstants.java

```diff
@@ -593,9 +594,10 @@ else if (man == 44) {
         Map<Integer, String> result = new HashMap<>();
         result.put(1, "no CAN bus, (firmware derived from CANCMD).");
         result.put(2, "integrated CANISB (firmware derived from CANCMD).");
-        result.put(3, "Isolated CANUSB and CBUS node.");
-        result.put(4, "(firmware derived from CANCMD).");
-        result.put(5, "System booster");
+        result.put(3, "integrated CANISB (firmware derived from CANCMD).");
+        result.put(4, "System booster");
+        result.put(4, "integrated CANISB (firmware derived from CANCMD).");
```

Comment:
Map value `4` has been defined twice and `5` skipped - is this intentional?

Reply:
Thanks for that.